src/tasks/batch_integration/metrics/clustering_overlap/script.py

```diff
@@ -15,28 +15,35 @@
 ## VIASH END
 
 print('Read input', flush=True)
-input = ad.read_h5ad(par['input_integrated'])
+input_solution = ad.read_h5ad(par['input_solution'])
+input_integrated = ad.read_h5ad(par['input_integrated'])
+
+input_solution.obsp["connectivities"] = input_integrated.obsp["connectivities"]
+input_solution.obsp["distances"] = input_integrated.obsp["distances"]
+
+# TODO: if we don't copy neighbors over, the metric doesn't work
+input_solution.uns["neighbors"] = input_integrated.uns["neighbors"]
```

Comment:
is it necessary to transfer the kNN graph to the solution? Wouldn't be sufficient to run on the integrated input only?

Reply:
We want to compare the integrated object's KNN graph to the solution's label, no? Because if we only compute the ARI and NMI on the data that is in the integrated object, then we don't know for certain that this anndata still has the same dimensionality as the solution.

Reply:
I would add the metadata to the integrated object instead. That way we can check that the dimensions match, but are essentially evaluating the integrated object. I think it's a matter of taste and would find it more intuitive to base things around the integrated object